doc/code/sf_cli.rst

```diff
@@ -0,0 +1,127 @@
+sf.cli
+======
+
+.. currentmodule:: strawberryfields.cli
+
+The Strawberry Fields command line program ``sf`` provides several utilities
+including:
+
+* ``sf configure [--token] [--local]``: configure the connection to the cloud platform
+
+* ``sf run input [--output FILE]``: submit and execute quantum programs from the command line
+
+* ``sf --ping``: verify your connection to the Xanadu cloud platform.
+
+
+Configure
+---------
+
+Using the ``sf configure`` command to create a configuration file on Linux:
+
+.. code-block:: bash
+
+    $ sf configure --token MYAUTH
+    $ cat  ~/.config/strawberryfields/config.toml
+    [api]
+    authentication_token = "MYAUTH"
+    hostname = "platform.strawberryfields.ai"
+    use_ssl = true
+    port = 443
+
+Using the ``sf configure`` command to create a configuration file on Windows:
+
+.. code-block:: ps1con
+
+    C:\> sf configure --token MYAUTH
+    C:\> cat C:\Users\USERNAME\AppData\Local\Xanadu\strawberryfields\config.toml

```

Comment:
Thanks, this looks pretty! :)

Reply:
Note, I couldn't get this to work! The powershell prompt is actually
```powershell
PS C:\> sf configure --token MYAUTH
```
but Sphinx gives a warning when I include the PS :( Also, the syntax highlighting doesn't seem to work either.

Reply:
I'll give this a look from the [simple command line of Windows](https://www.computerhope.com/issues/pictures/dos.jpg) and then maybe we could omit `PS`. :)

Reply:
Works fine with the standard command line as well! (Was using the Anaconda Prompt)

Reply:
Should the `ps1con` (which I assume stands for the 1st powershell console) be changed to `cmd` (or something similiar) then, since the prompt will be looking like that in cmd?

Reply:
Is there any particular reason why powershell is chosen as the "default" windows shell? Although I like powershell, I'm not sure if it's actually the default/most commonly used shell in Windows, but I might be wrong about that (it seems like Windows 10 set powershell as default a few years ago, but I'm not sure that's still the case since cmd is still very prevalent and powershell works quite a bit different from e.g. bash).

Reply:
Also, `cat` is only available in powershell (not in cmd), so that's another good reason for using that instead. 🙂

Reply:
Good points - let's change to `cmd` across the board